pages/common/join.md

```diff
@@ -6,6 +6,10 @@
 
 `join {{file1}} {{file2}}`
 
+- Join two files using a comma (instead of space) as a field separator:
+
+`join -t ',' {{file1}} {{file2}}`
+
 - Join field3 of file1 with field1 of file2:
 
 `join -1 3 -2 1 {{file1}} {{file2}}`
```

Comment:
While we are here, let's put 3 and 1 within tokens (`{{}}`). And same for 1 in the next example

Reply:
and the `','` on line 11 too?

Reply:
Yeah, good call.